uxarray/grid/grid.py

```diff
@@ -27,16 +29,12 @@
     _populate_edge_face_connectivity,
 )
 
-from uxarray.grid.coordinates import (
-    _populate_lonlat_coord,
-    _populate_cartesian_xyz_coord,
-)
-
```

Comment:
These shouldn't of been deleted.

Reply:
I am a little bit confused now. Based on the last conversation, you mentioned that the line 22, 23 should be deleted because they're unused. I checked, `_populate_lonlat_coord,
    _populate_cartesian_xyz_coord,` are being used by the `def node_x` and `def node_y`
    
And I checked, there're no duplicate calls to these two functions. I wonder if you can directly commit to the branch and show me what is your desired call? Thanks

Reply:
Took a look into the file and it all looks good, the git diff gave me the wrong impression at first.